rust/cyclotron-fetch/src/metrics_constants.rs

```diff
@@ -0,0 +1,15 @@
+pub const WORKER_SAT: &str = "fetch_worker_available_permits";
+pub const WORKER_DEQUEUED: &str = "fetch_worker_dequeued_jobs";
+pub const DEQUEUE_TIME: &str = "fetch_dequeue_ms";
+pub const SPAWN_TIME: &str = "fetch_spawn_tasks_ms";
+pub const JOB_TOTAL_TIME: &str = "fetch_job_total_run_ms";
+pub const JOB_INITIAL_REQUEST_TIME: &str = "fetch_job_initial_request_ms";
+pub const OUTCOME_LABEL: &str = "outcome";
+pub const RESPONSE_STATUS_LABEL: &str = "response_status";
+pub const BODY_FETCH_TIME: &str = "fetch_body_fetch_ms";
+pub const FETCH_JOB_ERRORS: &str = "fetch_job_errors";
+pub const FETCH_JOBS_COMPLETED: &str = "fetch_jobs_completed";
+pub const FETCH_DEAD_LETTER: &str = "fetch_dead_letter";
+pub const RESPONSE_RECEIVED: &str = "fetch_got_response";
+pub const BODY_FETCH_FAILED: &str = "fetch_body_fetch_failed";
+pub const BODY_FETCH_SUCCEEDED: &str = "fetch_body_fetch_succeeded";
```

Comment:
I like that the janitor has `cyclotron_` prefixes, since Prometheus metrics are a big global namespace we should probably do that here too.

Reply:
Done